### PR TITLE
HOTFIX - TMEDIA-199 - Fix mobile overflow issues

### PR DESCRIPTION
**Pull request by matthewroach**
## Description

Add CSS to set min and max width on advanced layout grid items for tablet and mobile breakpoints when code is using grid template areas and display: contents is in use which has width issues with some of the oembed/table layout

## Jira Ticket
- [TMEDIA-199](https://arcpublishing.atlassian.net/browse/TMEDIA-199)

## Test Steps

1. Checkout this branch `git checkout HOTFIX-TMEDIA-199-advanced-right-rail`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/right-rail-advanced-block`
3. Verify the test page scales down to mobile breakpoints without items causing horizontal scrolling http://localhost/2019/09/17/global-kitchen-sink-article/?_website=the-gazette&layout=right-rail-advanced

### Before

Overflow issues before - 
TMEDIA-199-before

### After

Page content scaled correctly - 
TMEDIA-199-after

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [X] Confirmed all the test steps a reviewer will follow above are working.
- [X] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [X] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [X] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - X ] Confirmed this PR has unit test files
  - [X] Ran `npm run test`, made sure all tests are passing
- [X] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

**Review by JackHowa — APPROVED** [review]
working as shown with stable vs hotfix

**Review by JackHowa — CHANGES_REQUESTED** [review]
we want to use hotfix tag first before merging. working as expected though

File: blocks/a11y-testing-block/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@wpmedia/a11y-testing-block",
-  "version": "5.10.2",
+  "version": "5.10.3-hotfix.0",
```
Comment (JackHowa) [inline blocks/a11y-testing-block/package.json:3]: interesting, it looks like this versioning updated the version of all the packages @matthewroach . what command did you use for this?

File: lerna.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "blocks/*",
     "components/*"
   ],
-  "version": "5.10.2",
+  "version": "5.10.3-hotfix.0",
```
Comment (JackHowa) [inline lerna.json:6]: ok cool, looks like lerna was used

**Review by JackHowa — APPROVED** [review]
good to go

File: lerna.json (hunk 2 above)
Comment (matthewroach) [reply lerna.json:6]: This was the command documented on the git docs in news theme development

Comment (JackHowa) [reply lerna.json:6]: oh ok cool https://github.com/WPMedia/fusion-news-theme-blocks/blob/stable/News%20Theme%20Development.md#publish-hotfix --> just checking

Comment (matthewroach) [reply lerna.json:6]: We need it to be `5.10.3` in `stable` - maybe the docs are a little outdated on hotfix steps, as we shouldn't have to bump the version for hotfix branch if we followed the same set up of `beta` or `canary` by using the commit hash

**Pull request merged**